bin/oref0-setup.sh

```diff
@@ -503,6 +505,10 @@ fi
 echo Running: openaps report add enact/suggested.json text determine-basal shell monitor/iob.json monitor/temp_basal.json monitor/glucose.json settings/profile.json $EXTRAS
 openaps report add enact/suggested.json text determine-basal shell monitor/iob.json monitor/temp_basal.json monitor/glucose.json settings/profile.json $EXTRAS
 
+# create .profile so that openaps commands can be executed from the command line
```

Comment:
It would be better if we could eliminate the need for these environment variables and have oref0 use the `ns` device instead. If you don't want to take that on, probably should add a TODO to remove this workaround when it's done.